README.md

```diff
@@ -53,10 +53,199 @@ Myget feeds:
 | Application Insights | [![MyGet Nightly][OpenTelemetry-exporter-ai-myget-image]][OpenTelemetry-exporter-ai-myget-url]                   | [![NuGet release][OpenTelemetry-exporter-ai-nuget-image]][OpenTelemetry-exporter-ai-nuget-url]                   |
 | Stackdriver          | [![MyGet Nightly][OpenTelemetry-exporter-stackdriver-myget-image]][OpenTelemetry-exporter-stackdriver-myget-url] | [![NuGet release][OpenTelemetry-exporter-stackdriver-nuget-image]][OpenTelemetry-exporter-stackdriver-nuget-url] |
 
-## OpenTelemetry QuickStart: collecting data
+## OpenTelemetry Tracing QuickStart: collecting data
 
-You can use OpenTelemetry API to instrument code and report data. Or use one of
-automatic data collection modules.
+You can use OpenTelemetry API to instrument code and report data.  Check out [Tracing API overview](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/overview.md) to learn more about distributed tracing.
+
+In the examples below we demonstrate how to create and enrich spans though OpenTelemetry API.
+
+OpenTelemetry also provides auto-collectors for ASP.NET Core, HttpClient calls (.NET Core) and Azure SDKs - [configuration](#configuration) section demonstrates how to enable it.
+
+### Obtaining tracer
+
+**Applications** should follow [configuration](#configuration) section to find out how to create/obtain tracer.
+
+**Libraries** must take dependency on OpenTelemetry API package only and should never instantiate tracer or configure OpenTelemetry. Libraries **will** be able to obtain *global* tracer that may either be noop (if user application is not instrumented with OpenTelemetry) or real tracer implemented in the SDK package.
+
+### Create basic span
+
+To create the most basic span, you only specify the name. OpenTelemetry SDK collects start/end timestamps, assigns tracing context and assumes status of this span is `OK`.
+
+```csharp
+var span = tracer
+    .SpanBuilder("basic span")
+    .StartSpan();
+
+span.End();
+```
+
+### Create nested spans
+
+In many cases you want to collect nested operations. You can propagate parent spans explicitly in your code or use implicit context propagation embedded into OpenTelemetry and .NET.
+
+#### Explicit parent propagation and assignment
+
+```csharp
+var parentSpan = tracer
+    .SpanBuilder("parent")
+    .StartSpan();
+
+var childSpan = tracer
+    .SpanBuilder("child")
+    .SetParent(parentSpan) // explicitly assigning parent here
+    .StartSpan();
+
+childSpan.End();
+parentSpan.End();
+```
+
+#### Implicit parent propagation and assignment
+
+```csharp
+var parentSpan = tracer
+    .SpanBuilder("parent")
+    .StartSpan();
+
+// calling WithSpan puts parentSpan into the ambient context
+// that flows in async calls.   When child is created, it
+// implicitly becomes child of current span
+using (tracer.WithSpan(parentSpan))
+{
+    var childSpan = tracer
+        .SpanBuilder("child")
+        .StartSpan();
+
+    childSpan.End();
+}

```

Comment:
Who calls End() on parentSpan? Or is not neededed as its done automatically when using ends.?

Reply:
dispose on Scope that WithSpan returns